src/Generation/GapicClientGenerator.php

```diff
@@ -361,25 +362,34 @@ private function lroMethods(): Vector
 
     private function getClientDefaults(): PhpClassMember
     {
+        $clientDefaultValues = [
+        'serviceName' => AST::access(AST::SELF, $this->serviceName()),
+        // TODO: Understand if this should be named 'serviceAddress' or 'apiEndpoint'?
+        'serviceAddress' => AST::concat(AST::access(AST::SELF, $this->serviceAddress()), ':', AST::access(AST::SELF, $this->servicePort())),
```

Comment:
nit: For readability can we split this up?

```php
$serviceAddress = AST::concat(
    AST::access(AST::SELF, $this->serviceAddress()),
    ':',
    AST::access(AST::SELF, $this->servicePort())
);
```

Reply:
Done.